LICENSE

```diff
@@ -21,4 +21,4 @@ OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
 ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE.
 
-For more information, please refer to <http://unlicense.org>
+For more information, please refer to <https://choosealicense.com/licenses/unlicense>
```

Comment:
I'm not sure I want to touch the license file itself because I don't know what are the legal implications for it... Maybe just change from `http` to `https`, as it appears here? https://choosealicense.com/licenses/unlicense

Reply:
Perhaps it is better to drop this fix as I am not familiar with licensing. I'll follow #247 instead and wait for the community.

Reply:
We could still merge the changes in the README.md file...

Reply:
@seladb I would say as the original unlicense give all the credit/copyright to anybody. Basically anyone can create a new repo and relicense this with another license without any issue as you have all credit/copyright without the needs to notice someone.